workers/data_refinery_workers/processors/transcriptome_index.py

```diff
@@ -133,7 +133,20 @@ def _handle_shell_error(job_context: Dict, stderr: str, command: str) -> None:
     job_context["success"] = False
 
 
-def _prepare_reference(job_context: Dict) -> Dict:
+def _create_index(job_context: Dict) -> Dict:
+    """Creates a salmon transcriptome index.
+
+    This index will only be appropriate for use in running salmon on
+    transcripts collected from an organism from the same species as
+    the Batch. Additionally it will either be a "long" index or a
+    "short" index, which means that it will be appropriate for reads
+    with a certain range of base pair lengths. The creator of Salmon,
+    the esteemed Rob Patro, has said:
+
+    "For *most* data (i.e. 75bp or longer), the default k should work
+    well.  For reads shorter than 75bp ... one should absolutely use a
+    shorter k (probably 23 or 21)."
```

Comment:
Can you add the source of this quotation?

Reply:
He said this in a slack channel. I can mention that's where it came from if you think that's a good idea. I can't actually link to the source of the quotation though which is why I included it here. Do you think I should state that it came from slack in the comment?

Reply:
Never mind then. I thought the source would be a URL. (In general I think it is a good idea to list the source in case readers want to dig deeper.)